contrib/mesos/pkg/executor/executor.go

```diff
@@ -179,6 +166,10 @@ func New(config Config) *KubernetesExecutor {
 	return k
 }
 
+func (k *KubernetesExecutor) InitialRegComplete() <-chan struct{} {
```

Comment:
Registered is part of the interface. InitiallyRegistered makes sense. But compare with the comment below. We can maybe get rid of this.